hbase-migration-tools/bigtable-hbase-replication/bigtable-hbase-replication-core/src/main/java/com/google/cloud/bigtable/hbase/replication/metrics/HBaseToCloudBigtableReplicationMetrics.java

```diff
@@ -24,4 +24,5 @@ private HBaseToCloudBigtableReplicationMetrics() {}
   public static final String INCOMPATIBLE_MUTATION_METRIC_KEY = "bigtableIncompatibleMutations";
   public static final String DROPPED_INCOMPATIBLE_MUTATION_METRIC_KEY =
       "bigtableDroppedIncompatibleMutations";
+  public static final String FUTURE_DELETE_MUTATION_METRIC_KEY = "bigtableFutureDeleteMutations";
```

Comment:
There is no delete happening "yet". The name should be bigtablePutsInFuture  or something like that.

Reply:
Renamed.